src/components/utils/rqb-inputs/directory-item-selector.js

```diff
@@ -0,0 +1,324 @@
+/**
```

Comment:
I think its a pitie to copy/paste most of the study-app version of this file. we hould consider to have a commons-ui version of this component.

Another way: there already is a "file selector" component in gridexplore : directory-selector.js.
In MR https://github.com/gridsuite/gridexplore-app/pull/358 I changed it to be able to pick up a Study.
Maybe you could have used it.

Reply:
I agree with David that we should fix this. Have you tried using the directory-selector or created a TS to avoid diverging implementations for too long?